features/fields_extensie.feature

```diff
@@ -1,126 +1,544 @@
+
+  Groepen kunnen in gezamenlijkheid worden gevraagd door de naam van de groep op te nemen in de fields-parameter. In dat geval worden alle attributen van de groep opgenomen in het antwoord, voor zover ze een waarde hebben. Bijvoorbeeld fields=burgerservicenummer,naam geeft naast het burgerservicenummer alle attributen van de naam (geslachtsnaam, voornamen, voorvoegsel, enz.) terug.
 
-  In de BRP API wordt de uitgebreide variant van fields ondersteund. Velden in een groep kunnen dus worden opgevraagd zonder het hele pad op te noemen, bijvoorbeeld fields=voornamen levert naam.voornamen.
+  Attributen binnen een groep kunnen ook individueel worden bevraagd. De dot-notatie wordt gebruikt om specifieke attributen van een groep te selecteren. Bijvoorbeeld de voornamen, geboortedatum en geboorteplaats van een persoon kunnen worden opgevraagd via fields=naam.voornamen,geboorte.datum,geboorte.plaats. Daarnaast kunnen attributen in een groep worden opgevraagd door alleen de naam van de property op te geven, of een deel van het pad, niet het hele pad ervoor. Dit kan nuttig zijn om de lengte van de waarde in fields te beperken.
+
+  Met de fields parameter kan een gevraagd veld worden aangeduid met:
+  - het hele pad naar dat veld, bijvoorbeeld fields=verblijfplaats.datumAanvangAdreshouding.jaar
+  - het laatste deel van het pad naar dat veld, bijvoorbeeld fields=datumAanvangAdreshouding.jaar
+  - alleen de veldnaam, bijvoorbeeld fields=jaar
+
+  Voor elk veld in fields moet dit leiden tot één uniek veld in de resource, ongeacht of dit veld bij de specifiek opgevraagde resource een waarde heeft. Bijvoorbeeld een persoon heeft geboorte.land en overlijden.land, dus is fields=land niet toegestaan. In dat geval wordt een foutmelding teruggegeven. In de foutmelding worden de gevonden velden teruggegeven in invalidParams.detail. Wanneer veel velden zijn gevonden (bijvoorbeeld meer dan 3), dan worden de eerste resultaten (bijv. de eerste 3 gevonden velden) gemeld.
+  Wanneer een veld in fields exact zo voorkomt in de resource, maar ook zo voorkomt als einde van het pad naar een ander veld, dan wordt het veld teruggegeven die exact wordt aangeduid met wat in fields is opgenomen.
 
   De volgende velden worden geleverd ongeacht of de fields parameter is gebruikt en ongeacht de eventueel gevraagde velden in de fields parameter:
   - inOnderzoek wordt meegegeven wanneer gevraagde corresponderende velden in onderzoek zijn. Zie in_onderzoek.feature voor uitleg wanneer attributen in onderzoek zijn.
   - geheimhoudingPersoonsgegevens wordt altijd meegegeven wanneer deze de waarde true heeft.
   - opschortingBijhouding reden en datum worden altijd meegegeven wanneer deze een waarde hebben.
```

Comment:
ik vind dat alleen opschortingBijhouding reden ongevraagd moet worden geleverd. opschortingBijhouding datum is niet voor iedereen relevant en bovendien zijn veel gebruikers niet voor opschortingBijhouding datum geautoriseerd

Reply:
Ik heb hier geen mening over en begrijp jou argumnent. Heb dit aangepast. 
@CathyDingemanse is dit ook hoe jij er tegenaan kijkt ?